cocos/network/CCDownloader-android.cpp

```diff
@@ -66,7 +66,8 @@ namespace cocos2d { namespace network {
         , _impl(nullptr)
         {
             // use local static variable make sure native methods registered once
-            static bool _registered = _registerNativeMethods(JniHelper::getEnv());
+            if(!_registered)
+                _registered = _registerNativeMethods(JniHelper::getEnv());
```

Comment:
I don't think we still need this, what we need is to guarantee the invocation of _preloadJavaDownloaderClass